OracleGoldenGate/23/Dockerfile

```diff
@@ -0,0 +1,30 @@
+## Copyright (c) 2024, Oracle and/or its affiliates.
+ARG           BASE_IMAGE=oraclelinux:8
+#  hadolint ignore=DL3006
+FROM        ${BASE_IMAGE}
```

Comment:
I don't know if this bothers you but this line is misaligned with the rest of the file. :)

Reply:
yes... but then the `BASE_IMAGE` variable name wouldn't align :)

Reply:
I hadn't even noticed that `BASE_IMAGE` was on both lines let alone that both instances were aligned. My brain took no notice of what was contained within the eye-twitch-inducing placement of `${...}` 😛 

It's your call either way, so I'll just have to gird my loins by putting on big boy pants if I ever need to review this `Dockerfile` again in future. 😂